src/components/navbar/navbar.component.tsx

```diff
@@ -4,18 +4,18 @@ import UserMenuPanel from "../navbar-header-panels/user-menu-panel.component";
 import SideMenuPanel from "../navbar-header-panels/side-menu-panel.component";
 import Logo from "../logo/logo.component";
 import { navigate, ExtensionSlot } from "@openmrs/esm-framework";
-import AppSwitcher20 from "@carbon/icons-react/lib/app-switcher/20";
+import Switcher20 from "@carbon/icons-react/lib/switcher/20";
```

Comment:
What's the difference?

Reply:
`AppSwitcher` icon is being deprecated got a console warning for that.